### ISSUE TITLE
Error after installation

### ISSUE THREAD

**Issue by Starraider** [christianfutterlieb/backend_roles#3]
After installing the extension, I get this error:
```
Symfony\Component\DependencyInjection\Exception\RuntimeException
Cannot autowire service "AawTeam\BackendRoles\Command\SynchronizeCommand": argument "$name" of method "__construct()" is type-hinted "string", you should configure its value explicitly.
```
I use TYPO3 11.5.30 and PHP 8.1.21

**Comment by jpmschuler** [comment]
Confirmed, poked it a bit, made it run. You could try that with composer-patches until the next release

**Comment by christianfutterlieb** [comment]
Thanks for you report, fixed the issue in d0fde015524b8ae010aed3f041999963f4170006, I'll release 2.0.2 shortly.
